.travis.yml

```diff
@@ -25,8 +25,7 @@ install:
   - chmod +x /tmp/urchin/package/urchin
   - '[ -z "$WITHOUT_CURL" ] || sudo apt-get remove curl -y'
 script:
-  - if [ -n "${SHELLCHECK-}" ]; then shellcheck -s bash nvm.sh && shellcheck -s sh nvm.sh && shellcheck -s dash nvm.sh && shellcheck -s ksh nvm.sh && shellcheck -s bash install.sh ; fi
-  - if [ -z "${SHELLCHECK-}" ]; then make TEST_SUITE=$TEST_SUITE URCHIN=/tmp/urchin/package/urchin test-$SHELL ; fi
+  - if [ -n "${SHELLCHECK-}" ]; then ./shellcheck.sh ; else make TEST_SUITE=$TEST_SUITE URCHIN=/tmp/urchin/package/urchin test-$SHELL ; fi
```

Comment:
There's 3 files - install, nvm, and nvm-exec.

Since travis.yml installs shellcheck, travis.yml should be the only place those commands live. If we can get it installable via `npm`, then i'd put it in `npm run-script`s in `package.json`.